onnx/numpy_helper.py

```diff
@@ -156,13 +155,9 @@ def to_list(sequence: SequenceProto) -> List[Any]:
     """
     lst: List[Any] = []
     elem_type = sequence.elem_type
-    value_field = mapping.STORAGE_ELEMENT_TYPE_TO_FIELD[elem_type]
-    values = getattr(sequence, value_field)
+    values = helper.get_attr_from_sequence_elem_type(sequence, elem_type)
```

Comment:
IMHO, I think it would be simpler to rewrite the code as below:
```py
   if elem_type == SequenceProto.TENSOR:
      return [to_array(v) for v in sequence.tensor_values]
   elif elem_type == SequenceProto.SPARSE_TENSOR:
      return [to_array(v) for v in sequence.sparse_tensor_values]
   elif elem_type == SequenceProto.SEQUENCE:
      return [to_list(v) for v in sequence.sequence_values]
   etc.
```
I don't think we need to define the helper methods like `get_attr_from_seqience_elem_type`.